src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -251,16 +255,22 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                     } else {
                         invalidErrList.addAll(validateEmailContactAndDomain(userRegistration));
                         if (invalidErrList.isEmpty()) {
-                            userRegistration.setUserAuthToken(inputDataMap.get(Constants.X_AUTH_TOKEN));
-                            String responseCode = userUtilityService.createBulkUploadUser(userRegistration);
-                            if (!responseCode.equalsIgnoreCase(Constants.OK)) {
-                                failedRecordsCount++;
-                                statusCell.setCellValue(Constants.FAILED_UPPERCASE);
-                                errorDetails.setCellValue(responseCode);
+                            if (StringUtils.isBlank(redisCacheMgr.getCache(userRegistration.getPhone()+ "" +userRegistration.getEmail()))) {
+                                redisCacheMgr.putCache(userRegistration.getPhone().trim()+ "" +userRegistration.getEmail(),
+                                        userRegistration);
```

Comment:
No need to put the data -- only key is sufficient -- just check the key exists

Reply:
Done